sickbeard/server/web/core/base.py

```diff
@@ -91,6 +91,10 @@ def __init__(self, rh, filename):
             'newsBadge': '',
             'toolsBadge': '',
             'toolsBadgeClass': '',
+            'base_url': urljoin(rh.request.protocol + '://' + rh.request.host,
+                                sickbeard.WEB_ROOT) +
+                          ('', '/')
+                          [urljoin(rh.request.protocol + '://' + rh.request.host, sickbeard.WEB_ROOT)[-1] not in '/'],
```

Comment:
I'm not sure why you do this. WEB_ROOT can't have an ending slash, so why don't we just always add it?
For example:
```urljoin(rh.request.protocol + '://' + rh.request.host, sickbeard.WEB_ROOT + '/')```